ecllibrary/std/OpenSSL.ecl

```diff
@@ -0,0 +1,329 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2025 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the License);
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an AS IS BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+EXPORT OpenSSL := MODULE
+
+IMPORT lib_openssl;
+
+EXPORT Digest := MODULE
+
+    /**
+     * Returns a list of the names of the available hash digest algorithms.
```

Comment:
Add the "this is an introspection function" paragraph here. You can crib a version of it from other functions where it is already included.

Reply:
Fixed.